src/features/user/Widget/DonationLink/CustomSnackBar.tsx

```diff
@@ -0,0 +1,47 @@
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
+import i18next from '../../../../../i18n';
+import { ReactElement } from 'react';
+import styled from '@emotion/styled';
+
+interface SnackBarProps {
+  SnackBartext: String;
```

Comment:
follow camelcase convention for variable names i.e. this should be `snackBarText`